WalletWasabi/Wabisabi/IssuanceRequest.cs

```diff
@@ -11,8 +14,14 @@ internal IssuanceRequest(GroupElement ma, IEnumerable<GroupElement> bitCommitmen
 			BitCommitments = bitCommitments;
 		}
 
+		/// <summary>
+		/// Pedersen commitment to the credential amount.
+		/// </summary>
 		public GroupElement Ma { get; }
 		
+		/// <summary>
+		/// Commitments the the value's bits decomposed amount.
```

Comment:
This comment should be fixed.

Reply:
Maybe: "A list of Pedersen commitments to the credential amount's binary decomposition."